windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -44,6 +45,31 @@ wchar_t * getStr(int strId) {
 	return finalStr;
 }
 
+char * getKolibriLinkAddress() {
+	/*
+	This will return the current Kolibri running link url.
+	It will fetch the PORT value from server.pid file.
+	*/
+	char * kolibriHomeEnv = getenv("KOLIBRI_HOME");
+	if (kolibriHomeEnv == NULL) {
+		kolibriHomeEnv = "";
+	}
+	char * configName = "\\server.pid";
+	char * pidFile = joinChr(kolibriHomeEnv, configName);
+	const char * port;
+	std::ifstream file(pidFile);
+	if (file.is_open()) {
+		std::string line;
+		while (getline(file, line)) {
+			port = line.c_str();
+		}
+		file.close();
+		char * httpLink = joinChr("http://127.0.0.1:", line.c_str());

```

Comment:
Could we do some validation here to make sure that `line.c_str()` actually is a port?

Reply:
@indirectlylit Okay i'll validate it.